package.json

```diff
@@ -26,6 +26,7 @@
     "eslint-plugin-react": "^7.10.0",
     "file-loader": "^2.0.0",
     "font-awesome": "^4.4.0",
+    "iframe-resizer": "^3.6.5",
```

Comment:
looks like v4 was published not that long ago.  You should update to it :)

Reply:
updated here and on the other branch